src/Microsoft.Diagnostics.Runtime/src/Common/ClrField.cs

```diff
@@ -43,9 +43,9 @@ public abstract class ClrField
         public virtual bool IsPrimitive => ElementType.IsPrimitive();
 
         /// <summary>
-        /// Gets a value indicating whether this field is a ValueClass (struct).
+        /// Gets a value indicating whether this field is a value type.
         /// </summary>
-        /// <returns>True if this field is a ValueClass (struct), false otherwise.</returns>
+        /// <returns>True if this field is a value type, false otherwise.</returns>
         public virtual bool IsValueClass => ElementType.IsValueClass();
```

Comment:
We should probably rename IsValueClass (and all related instances) to ValueType since that's what the class is called? 
 [ValueType](https://docs.microsoft.com/en-us/dotnet/api/system.valuetype?view=netframework-4.8).

Reply:
That's my thought. I guess this was named `ValueClass` because there is an `IsValueClass` API in some `ICorDebug*` interface?

Reply:
Yeah it's from old diagnostics code and the term made its way into ClrMD ages ago.  I'll file an issue to remember to do it.